### PR TITLE
Update another.c

### PR DESCRIPTION
**Pull request by boundvar**
foo

### CODE REVIEW

File: another.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+extern int some_constant = 1;
```
Comment (vi117) [inline another.c:3]: Upper Case.
`SOME_CONSTANT`

File: another.c [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+extern int some_constant = 1;
+
+void foo(){
+  printf("foo!");
+}
```
Comment (vi117) [inline another.c:7]: blar blar blar

**Review by vi117 — CHANGES_REQUESTED** [review]
ccc

File: another.c (hunk 1 above)
Comment (boundvar) [reply another.c:3]: code style UPPER CASE

**Review by vi117 — APPROVED** [review]
good

**Pull request merged**